builtin/providers/aws/resource_aws_cloudformation_stack.go

```diff
@@ -143,7 +149,7 @@ func resourceAwsCloudFormationStackCreate(d *schema.ResourceData, meta interface
 	wait := resource.StateChangeConf{
 		Pending:    []string{"CREATE_IN_PROGRESS", "ROLLBACK_IN_PROGRESS", "ROLLBACK_COMPLETE"},
 		Target:     []string{"CREATE_COMPLETE"},
-		Timeout:    30 * time.Minute,
+		Timeout:    time.Duration(int64(timeout)) * time.Minute,
```

Comment:
It would have been cleaner to start with `int64` variable instead of casting it later.

Not that I would be worried about _minutes_ going over 32bits, but strictly speaking we might potentially end up comparing int32 with int64 in the condition above.

i.e. this 
```go
timeout := 30
```
can be turned into
```go
retryTimeout := int64(30)
```
and the line here can be turned into 
```go
Timeout:    time.Duration(retryTimeout) * time.Minute,
```

--- 

as hinted above, I'd also suggest renaming the `timeout` variable to more specific `retryTimeout`.